benchmark/go-pq-cdc-kafka/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/Trendyol/go-pq-cdc/benchmark/go-pq-cdc-kafka
 
-go 1.22.4
```

Comment:
We are maintaining version and dependency upgrades separately. Could you please revert these changes from this feature?